napari/_qt/layer_controls/qt_points_controls.py

```diff
@@ -187,7 +187,7 @@ def mouseMoveEvent(self, event):
         self.layer.status = self.layer.mode
 
     def _on_mode_change(self, event):
-        """"Update ticks in checkbox widgets when points layer mode is changed.
+        """ "Update ticks in checkbox widgets when points layer mode is changed.
```

Comment:
manual fix

Reply:
it means remove additional `"`?

Reply:
yeah, this should just be 3 of the `"` and get rid of the extra one at beginning/ end. was a typo initially